2023/fortune_obadiah

```diff
@@ -0,0 +1,136 @@
+Hello, World! I'm Fortune Obadiah, hailing from Delta State, Nigeria, and currently residing in Eket, Akwa Ibom. My journey into the world of tech began just three months ago when I embarked on the exciting path of a "Data Enthusiast," fueled by my deep love for all things data.
+
+Before diving into data science, my tech journey started last year as a digital marketer and content writer. I've been honing my skills at Africinnovate, a fantastic tech hub right here in Akwa Ibom, which has been a tremendous source of support for my burgeoning tech career.
+
+My motivation to contribute to the Outreachy project with Wagtail stems from a heartfelt belief that accessibility on the internet should be an inclusive experience for everyone, including those with disabilities. It's my firm conviction that no one should face unnecessary obstacles when navigating websites and the online world.
+
+This marks my maiden voyage into the world of open source, and I couldn't be more thrilled. Securing an internship with Outreachy would be a dream come true, aligning perfectly with my birthday wishes this year - to give back to the community. 🎂✨
+
+Outside of my tech pursuits, I have a creative side where I indulge in writing poetry, crafting spoken words, and penning song lyrics. I'm also an avid history buff who relishes diving into documentaries and curling up with a good book 📖🎵. In my free time, I enjoy reading, singing, and experimenting with new recipes. 🍳🎤
+
+I am enthusiastic and fully prepared to roll up my sleeves and immerse myself in the Outreachy community, ready to work hard and learn from the best. Let's make this journey a memorable one! 🚀🌟
+
+|    Social_handles     |                Link                                      |
+|----------------------|-------------------------------------------|
+| GitHub Profile       | (https://github.com/triumphant-fortune) |
+| Medium               |   [(https://medium.com/@fortuneweb3info)] |
+| LinkedIn             | [(https://www.linkedin.com/in/smartfortune-obadiah-895932264/) |
+| Twitter              | [(https://twitter.com/OMGitsfortune1)]   |
+
+
+##Tasks
+
+[#Outreachy contributor guide]
```

Comment:
This kind of formatting isn’t going to work too well in Markdown! If in doubt make sure to preview your writing before committing it.

Reply:
@thibaudcolas Thanks for your feedback. I'll improve the Markdown formatting and remember to preview it before committing.